addons/medical/stringtable.xml

```diff
@@ -5623,5 +5623,23 @@
             <Chinesesimp>关闭盖子</Chinesesimp>
             <Korean>뚜껑 닫기</Korean>
         </Key>
+        <Key ID="STR_ACE_Medical_MedicalSettings_allowDeadBodyMovement_DisplayName">
+            <English>allowDeadBodyMovement</English>
+        </Key>
+        <Key ID="STR_ACE_Medical_MedicalSettings_allowDeadBodyMovement_Description">
+            <English>Not referenced anywhere in the source, does nothing ?!</English>
```

Comment:
Why add a translation then if it doesn't "exist"?

Reply:
To alert someone with more knowledge of the settings that it might be deptricated.

Reply:
That setting should just be removed (commented out for now at least).

Reply:
Commented out and entries removed from `stringtable.xml`. Did a testbuild and doesn't seem to have broken anything.